src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -650,7 +650,7 @@ static void ViewportSurfaceDrawTileSideBottom(
 
     neighbourCornerHeight1 = cornerHeight2;
 
-    for (uint32_t tunnelIndex = 0; tunnelIndex < TUNNEL_MAX_COUNT;)
+    for (uint8_t tunnelIndex = 0; tunnelIndex < kTunnelMaxCount;)
```

Comment:
Better to use `auto` in this case. If the type ever changes, this will be automatically updated.

Reply:
Fix in #21760